### ISSUE TITLE
A Ruqqus UWP app for Windows 10?

### ISSUE THREAD

**Issue by SFM61319** [ruqqus/ruqqus#295]
The title says it all.  
  
The procedure in the documentation is for Mac (macOS), and not for Windows. After making Ruqqus better than now (like the final stable stage for all public, usable by anyone, anywhere), can we get a Ruqqus ***UWP*** app for ***Windows 10*** (with like `AcrylicBrush` and `RevealBrush`)? (I would love to contribute to it as much as I can, if that's ever gonna happen)  
  
That would be great for those who don't wanna open Ruqqus on their browser or their phone.  
  
Thank you!

**Comment by jfwerner** [comment]
Doesn't sound bad actually. I can't think of a single social media really having a desktop program. Do you know how many people would actually use that?

For now I believe a lot of the ruqqus community is more on the power user side, but as with reddit, most users will be on mobile. I  don't know wether a desktop client is worth the effort.

**Comment by SFM61319** [comment]
> ..but as with Reddit, most users will be on mobile  
  
That is because Reddit doesn't have a UWP app (forget UWP, Reddit doesn't have a local desktop client version on **ANY** OS). And if I remember correctly, [Instagram has a win10 app](https://www.microsoft.com/store/productId/9NBLGGH5L9XT "Instagram - Windows 10")

**Comment by abysmor** [comment]
I would also like to suggest Electron as another way to create a universal
app. Plus side is that it would be multi-platform and wouldn't require
crazy refactoring (Mac, Linux, and Windows all support Electron natively
since it is HTML/CSS/JS with some Java thrown in as well). I know some
people don't like Electron but it's crazy versatile.

*Elijah Ah-Sha*

**Comment by SFM61319** [comment]
@abysmor Or even React works fine because it uses the platforms default ui elements and works really good on mobiles too (without messing with the device ui consistency - unlike win10 where no two built-in microsoft apps look the same/have the same ui)

**Comment by jfwerner** [comment]
> And if I remember correctly, Instagram has a win10 app

They do, and it's abysmal. Really bad user experience.

> I would also like to suggest Electron as another way to create a universal
> app.
God please, no. We're trying to decrease power to big tech, not increase it. Google needs to die, and electron applications are just so ressource wasteful.

The ruqqus website is fast and light, I don't think there's a big need for a desktop client. I think we should let the community develop 3rd party clients, so similar to reddit people can choose their prefered app.

**Comment by SFM61319** [comment]
> The ruqqus website is fast and light, I don't think there's a big need for a desktop client. I think we should let the community develop 3rd party clients, so similar to reddit people can choose their prefered app.  
  
I'll try making one, then.  
  
But then I need some details (which I can search for online) and Ruqqus logo pics (PNGs are preferable) for the app logos and splash screen logos, etc., can you please provide me with PNG logos of Ruqqus ([similar to this](https://ruqqus.com/assets/favicon.ico "Browser logo")), possibly in 4k (or even HD), @ruqqus, @captainmeta4, @kek7198 or anyone else?

**Comment by AgeOfMarcus** [comment]
@SFM61319 I've managed to create a svg of the Ruqqus icon. It's in black and white, but that can be changed. You should be able to resize it to whatever you want without losing quality https://gofile.io/d/IvgV0Q

**Comment by SFM61319** [comment]
@AgeOfMarcus I really appreciate that! But that icon is *similar* to the actual icon, and not the same. I don't know if I'll be hit with copyrights and stuff. Also, using actual/original icons gives a feeling of originality. But I'll use your icons until they provide us with the actual icons. And if they don't, then I'll continue using yours. Again, thanks a lot for those!

**Comment by AgeOfMarcus** [comment]
@SFM61319 I just used an ico to SVG converter online

**Comment by SFM61319** [comment]
@AgeOfMarcus Idk man they looked different to me. Lemme check again.

**Comment by SFM61319** [comment]
@AgeOfMarcus wait, where did you get an ICO? I only found SVGs and PNGs in the [Assets](https://github.com/ruqqus/ruqqus/tree/master/ruqqus/assets) folder!

**Comment by AgeOfMarcus** [comment]
@SFM61319 just from the websites favicon.ico linked above lol, but I'm sure all those images can be converted too just search "ico to svg online" in google

**Comment by SFM61319** [comment]
@AgeOfMarcus They looked different to me because of the midpoint of the letter `r` in the icon. Can you see the bulge in the middle or am I the only one who is seeing one?

**Comment by AgeOfMarcus** [comment]
@SFM61319 to be honest I'm on mobile rn and it's not even letting me redownload the svg so I have no clue but all I did was run it though one of those sites

**Comment by SFM61319** [comment]
@AgeOfMarcus I understand that but what I don't understand is where did you get the ICO icon? Because there are no ICOs in the whole [ruqqus repo](https://github.com/ruqqus/ruqqus) (as far as I know). Because ICOs are used for PCs and there is no PC client for ruqqus yet.

**Comment by AgeOfMarcus** [comment]
@SFM61319 https://ruqqus.com/assets/favicon.ico

**Comment by SFM61319** [comment]
@AgeOfMarcus Error 1011 - Access denied  
  
Also, that's ruqqus' website, i only searched the ruqqus repo, so i might've missed it

**Comment by AgeOfMarcus** [comment]
@SFM61319 that's very strange, I'm not sure why you're getting that error. But anyway in the ruqqus repo there's some svg's in ruqqus/assets/logo

**Comment by SFM61319** [comment]
@AgeOfMarcus yeah I saw those SVGs, but they were not opening on github for some reason.  
  
I also mentioned seeing SVGs in [this comment](#issuecomment-663680816)
> @AgeOfMarcus wait, where did you get an ICO? I only found SVGs and PNGs in the Assets folder!

**Comment by AgeOfMarcus** [comment]
@SFM61319 sorry I meant here https://www.github.com/ruqqus/ruqqus/tree/master/ruqqus%2Fassets%2Fimages%2Flogo

But I don't know, they're all opening for me, and I just found that one in my browser when it loads the ruqqus icon

**Comment by SFM61319** [comment]
@AgeOfMarcus that is the same folder i am talking about. Take [this](https://github.com/ruqqus/ruqqus/blob/master/ruqqus/assets/images/logo/ruqqus_logo_square_white.svg), for example, is it loading properly on your phone? Because all I see some weird icon barely a few millimeters wide and tall.

**Comment by AgeOfMarcus** [comment]
@SFM61319 oh you're right, it doesn't load for me either

**Comment by SFM61319** [comment]
@AgeOfMarcus The actual image (actual/intrinsic dimensions):  
![The processed SVG on PC](https://raw.githubusercontent.com/ruqqus/ruqqus/dfab039adf505000afe4b0e4eeeffb3fc568b499/ruqqus/assets/images/logo/ruqqus_logo_square_white.svg)
  
[This](https://raw.githubusercontent.com/ruqqus/ruqqus/dfab039adf505000afe4b0e4eeeffb3fc568b499/ruqqus/assets/images/logo/ruqqus_logo_square_white.svg) is what I am seeing:  
<img alt="2 x 2 px actual shown image" src="https://raw.githubusercontent.com/ruqqus/ruqqus/dfab039adf505000afe4b0e4eeeffb3fc568b499/ruqqus/assets/images/logo/ruqqus_logo_square_white.svg" height="4px" width="4px" />

**Comment by SFM61319** [comment]
@AgeOfMarcus and none of them is the actual SVG icon.
